py/selenium/webdriver/chrome/command.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2014 Software Freedom Conservancy
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium.webdriver.remote.command import Command
+
+class ChromeCommand(Command):
+   """
+   Defines Chrome specific commands while including standard WebDriver ones.
+   """
+   LAUNCH_APP = "launchApp"
```

Comment:
Enums isn't really a thing in Python.  Please just embed the "launchApp" string in chrome/webdriver.py.